frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "dev": "razzle start",
     "build": "razzle build",
     "heroku-postbuild": "razzle build",
+    "dbg": "node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
what does this do?

Reply:
This is a handy shortcut for using the node debugger.